sapi/fuzzer/fuzzer-sapi.c

```diff
@@ -294,9 +294,11 @@ int fuzzer_do_request_from_buffer(
 void fuzzer_call_php_func_zval(const char *func_name, int nargs, zval *args) {
 	zval retval, func;
```

Comment:
You still have the func variable, which is still zval_ptr_dtor'ed at the end. That will now operate on uninitialized memory because it's never filled in / used.

Reply:
Good catch